docs/_includes/ds6/menu.html

```diff
@@ -254,8 +253,60 @@ <h3 id="menu-growth">Growth Menu</h3>
         </div>
     </div>
 </span>
-    {% endhighlight %}
+        {% endhighlight %}
+    </div>
+
+    <p>TIP: Use the
+        <span class="highlight">menu__items--grow-reverse</span> or
+        <span class="highlight">fake-menu__items--grow-reverse</span> element modifier to grow the width of the menu overlay in reverse direction.</p>
 
-    <p>TIP: Use the <span class="highlight">menu__items--grow-reverse</span> or <span class="highlight">fake-menu__items--grow-reverse</span> element modifier to grow the width of the menu overlay in reverse direction.</p>
+    <h3 id="menu-borderless">Borderless Menu</h3>
+    <p>Use the <span class="highlight">btn--borderless</span> element modifier on the expand button
+        to allow for a borderless version of the expand button.</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <span class="menu">
+                <button class="expand-btn btn--borderless" aria-haspopup="true" type="button">
```

Comment:
The bass class is `expand-btn`, so it's modifier has to be `expand-btn--borderless`.

Reply:
You know, I actually had it that way at the very beginning. I’ll adjust.